.github/workflows/ci.yml

```
@@ -551,6 +551,19 @@ jobs:
         env:
           RUST_BACKTRACE: full
         run: cargo nextest run --locked --release -p sway-lsp --no-capture --profile ci --config-file sway-lsp/tests/nextest.toml
+  cargo-test-forc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ env.RUST_VERSION }}
+      - uses: Swatinem/rust-cache@v2
+      - name: Run forc tests sequentially
+        env:
+          RUST_BACKTRACE: full
+        run: cargo test --locked --release -p forc -- --nocapture --test-threads=1
```

```suggestion
        run: cargo test --locked --release -p forc -- --nocapture
```

Is there any reason we need to run these tests with just 1 thread? LSP was doing this as the amount of memory the runner was using was getting too high when tests were running concurrently.